### ISSUE TITLE
Migration to official ed25519

### ISSUE THREAD

**Issue by sah4ez** [sah4ez/now#50]
Need to migrate from [this](https://github.com/sah4ez/pspk/blob/master/pkg/keys/keys.go#L10) to [this](https://godoc.org/golang.org/x/crypto/ed25519), because `github.com/agl/ed25519` unmaintainable.

**Comment by sah4ez** [comment]
@marti1125 Hello! If you have the opportunity to take this issue.

**Comment by marti1125** [comment]
sure I will check that :) :+1:

**Comment by marti1125** [comment]
assign to me :)

**Comment by marti1125** [comment]
Hi! the last PR I updated the method Verify. We need to update all methods ? :confused:

**Comment by sah4ez** [comment]
@marti1125 Right, but I have begun the big refactoring for usage a wasm... So if You have an opportunity, you can do PR to [the feature branch](https://github.com/sah4ez/pspk/tree/feature/webasm)
